src/components/Nav/index.js

```diff
@@ -166,6 +166,11 @@ const Nav = ({ handleThemeChange, isDarkTheme }) => {
           to: "/what-is-ethereum/",
           shouldDisplay: contentVersion > 1,
         },
+        {
```

Comment:
@ryancreatescopy should this be the 2nd item under individuals? I'd make the case it should be at the bottom.